robottelo/ui/factory.py

```diff
@@ -361,6 +362,42 @@ def make_hostgroup(session, org=None, loc=None, force_context=True, **kwargs):
     Hostgroup(session.browser).create(**create_args)
 
 
+def make_host(session, org=None, loc=None, force_context=True, **kwargs):
+    """Creates a host"""
+
+    create_args = {
+        u'arch': None,
+        u'cpus': '1',
+        u'cv': None,
+        u'custom_ptable': None,
+        u'domain': None,
+        u'env': None,
+        u'host_group': None,
+        u'ip_addr': None,
+        u'lifecycle_env': None,
+        u'loc': None,
+        u'mac': None,
+        u'media': None,
+        u'memory': '768 MB',
+        u'name': None,
+        u'org': None,
```

Comment:
Not sure if I understand it correctly but these are `make host 's` arguments. We use same arguments as defined with `create` function.
 https://github.com/SatelliteQE/robottelo/blob/master/robottelo/ui/hosts.py#L97